README.md

```diff
@@ -32,7 +46,7 @@ Becareful, some commands as no effect on iOS because Apple don't let us to do wh
 :question:(4) : I think there is a way to get the IP address but for now, this is not implemented..
 
 :warning:(5): Wifi API changes in Android SDK >= 29, restricts certain behaviour:
-  * a. Enable/Disable Wifi Module is deprecated and will always fail (except DO, PO and system apps) [[docs](https://developer.android.com/reference/android/net/wifi/WifiManager#setWifiEnabled(boolean))].
+  * a. Enable/Disable Wifi Module is deprecated and will always fail [[docs](https://developer.android.com/reference/android/net/wifi/WifiManager#setWifiEnabled(boolean))]. If you  want to open "Wifi Setting" in that case then, set the `shouldOpenSettings: true` when calling `setEnabled`.
```

Comment:
@mavyfaby I have added `shouldOpenSettings` option here in docs, instead of new section.

Give me your views regarding it.

Reply:
That's great then. My appologies😊